### PR TITLE
move the scenarios section up with the use cases section

### LINKED ISSUE #1101: move sample scenarios section up to be after use cases section
this would be just a cut'n'paste to move the entire unchanged [#sctn-sample-scenarios](https://w3c.github.io/webauthn/#sctn-sample-scenarios) section to be after [#sctn-use-cases](https://w3c.github.io/webauthn/#sctn-use-cases)

### PR DESCRIPTION
**Pull request by equalsJeffH**
fixes #1101 

since this is a whole-sale-move of a substantial section of the spec source from one place (near the end) to another (near the beginning), it seems bikeshed's diff is confused and not terribly useful.  If you really want to review, it seems best to use kdiff3 locally (the latter handled this just fine for me) and maybe other local diff tools.  or just build the spec and look at it :)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1242.html" title="Last updated on Jun 26, 2019, 10:48 PM UTC (a6c1a98)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1242/268fff6...a6c1a98.html" title="Last updated on Jun 26, 2019, 10:48 PM UTC (a6c1a98)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -422,1025 +422,901 @@ A variety of additional use cases and configurations are also possible, includin
...
+1. The [=client platform=] searches for and locates the authenticator.
 
-All [=CBOR=] encoding performed by the members of the above conformance classes MUST be done using the
-[=CTAP2 canonical CBOR encoding form=].
-All decoders of the above conformance classes SHOULD reject CBOR that is not validly encoded
-in the [=CTAP2 canonical CBOR encoding form=] and SHOULD reject messages with duplicate map keys.
+1. The [=client=] connects to the authenticator, performing any pairing actions if necessary.
 
+1. The authenticator shows appropriate UI for the user to select the authenticator on which the new credential will be
+    created, and obtains a biometric or other authorization gesture from the user.
```
Comment (emlun) [inline index.bs]: This looks inaccurate - maybe it used to say the "the _client_ shows appropriate UI..."? Should perhaps be more like:

>The authenticator shows appropriate UI for the user to provide a biometric or other authorization gesture.

File: index.bs [hunk 2]
```diff
@@ -422,1025 +422,901 @@ A variety of additional use cases and configurations are also possible, includin
...
+1. The user enters a username and password to log in. After successful login, the user is redirected back to example.com.
 
+1. The [=[RP]=] script runs the code snippet below.
 
-# Terminology # {#sctn-terminology}
+    1. The user agent checks if a [=user-verifying platform authenticator=] is available. If not, terminate this flow.
 
-: <dfn>Assertion</dfn>
-:: See [=Authentication Assertion=].
+    1. The [=[RP]=] asks the user if they want to create a credential with it.
```
Comment (emlun) [inline index.bs]:
```suggestion
    1. The [=[RP]=] asks the user if they want to create a credential with it. If not, terminate this flow.
```

File: index.bs [hunk 3]
```diff
@@ -422,1025 +422,901 @@ A variety of additional use cases and configurations are also possible, includin
...
 
-: <dfn>Attestation</dfn>
-:: Generally, <em>attestation</em> is a statement serving to bear witness, confirm, or authenticate.
-    In the WebAuthn context, [=attestation=] is employed to <em>attest</em> to the <em>provenance</em> of an [=authenticator=]
-    and the data it emits; including, for example: [=credential IDs=], [=credential key pairs=], signature counters, etc. An
-    [=attestation statement=] is conveyed in an [=attestation object=] during [=registration=]. See also [[#sctn-attestation]]
-    and [Figure 5](#fig-attStructs). Whether or how the [=client=] conveys the [=attestation statement=] and [=AAGUID=]
-    portions of the [=attestation object=] to the [=[RP]=] is described by [=attestation conveyance=].
+    1. If no [=user-verifying platform authenticator=] is available,
```
Comment (emlun) [inline index.bs]:
```suggestion

```

File: index.bs [hunk 4]
```diff
@@ -422,1025 +422,901 @@ A variety of additional use cases and configurations are also possible, includin
...
+        or if the user does not want to create a credential, terminate this flow.
```
Comment (emlun) [inline index.bs]:
```suggestion

```

File: index.bs [hunk 5]
```diff
@@ -422,1025 +422,901 @@ A variety of additional use cases and configurations are also possible, includin
...
+The following are possible situations in which decommissioning a credential might be desired. Note that all of these are
+handled on the server side and do not need support from the API specified here.
 
-        :   <dfn>rpId</dfn>
-        ::  The [=Relying Party Identifier=], for the [=[RP]=] this [=public key credential source=] is [=scoped=] to.
+- Possibility #1 -- user reports the credential as lost.
+    * User goes to server.example.net, authenticates and follows a link to report a lost/stolen [=authenticator=].
+    * Server returns a page showing the list of registered credentials with friendly names as configured during registration.
+    * User selects a credential and the server deletes it from its database.
+    * In future, the [=[RP]=] script does not specify this credential in any list of acceptable credentials, and assertions
```
Comment (emlun) [inline index.bs]:
```suggestion
    * In the future, the [=[RP]=] script does not specify this credential in any list of acceptable credentials, and assertions
```

File: index.bs (hunk 1 above)
Comment (equalsJeffH) [reply index.bs]: OK.

**Comment by equalsJeffH** [comment]
> I found some opportunities for improvements, 

ok, great :)

> but let's handle these in a separate PR to keep the diff manageable.

ok, so shall we just go ahead and merge this and then you will submit an issue wrt desiring to perform the above polishing?

**Comment by emlun** [comment]
Yep, that sounds good to me. :)

**Pull request merged**